multimap/src/test/java/org/infinispan/multimap/impl/SuperUser.java

```diff
@@ -0,0 +1,45 @@
+package org.infinispan.multimap.impl;
+
+import static java.util.Objects.hash;
+
+import java.io.IOException;
+import java.io.ObjectInput;
+
+import org.infinispan.commons.marshall.SerializeWith;
+
+@SerializeWith(SuperUser.SuperUserExternalizer.class)
+public class SuperUser extends User {
```

Comment:
The comment I got in JDG was that I created a User, but in reality we not using the "age" in the tests. So there is already a bean in tests called "Person" that serves the same purpose. 
Would you mind to do the modification in the PR ? It's not a bug deal, I would do it when this branch will be integrated in infinispan -hopefully soon :)

Reply:
Right, forgot about that in the morning stupor, swapped